src/internal.c

```diff
@@ -33340,134 +33340,93 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
 #endif /* !WOLFSSL_NO_TLS12 */
 
 #ifdef HAVE_SESSION_TICKET
+    /* Make a work from the front of random hash */
+    static WC_INLINE word32 MakeWordFromHash(const byte* hashID)
+    {
+        return ((word32)hashID[0] << 24) | ((word32)hashID[1] << 16) |
+               ((word32)hashID[2] <<  8) |  (word32)hashID[3];
+    }
 
-#define WOLFSSL_TICKET_FIXED_SZ (WOLFSSL_TICKET_NAME_SZ + \
-                WOLFSSL_TICKET_IV_SZ + WOLFSSL_TICKET_MAC_SZ + OPAQUE32_LEN)
-
-#if defined(WOLFSSL_GENERAL_ALIGNMENT) && WOLFSSL_GENERAL_ALIGNMENT > 0
-    /* round up to WOLFSSL_GENERAL_ALIGNMENT */
-    #define WOLFSSL_TICKET_ENC_SZ \
-        (((SESSION_TICKET_LEN - WOLFSSL_TICKET_FIXED_SZ) + \
-            WOLFSSL_GENERAL_ALIGNMENT - 1) & ~(WOLFSSL_GENERAL_ALIGNMENT-1))
-#else
-    #define WOLFSSL_TICKET_ENC_SZ (SESSION_TICKET_LEN - WOLFSSL_TICKET_FIXED_SZ)
-#endif
+    /* Check to make sure that the callback has actually encrypted the ticket */
+    static word32 compute_InternalTicket_hash(InternalTicket *a)
```

Comment:
Can you please change this function name? `compute_InternalTicket_hash` -> `ComputeInternalTicketHash`?

Reply:
Changed.

Reply:
Refactored into `HashObject`.